so3/kernel/process.c

```diff
@@ -608,8 +608,10 @@ void load_process(elf_img_info_t *elf_img_info)
 {
 	unsigned long section_start, section_end;
 	unsigned long segment_start, segment_end;
-	int i, j, k;
+	int i, j, k, l;
 	bool section_supported;
+	char section_base_name[16];
+	int section_name_dots;
 
 	/* Loading the different segments */
 	for (i = 0; i < elf_img_info->header->e_phnum; i++)
```

Comment:
put the `{` at same line as the `for` (Linux convention)

Reply:
Done